testing/python_lib/build_config.py

```diff
@@ -246,7 +265,8 @@ def main(argv):
     else:
         raise Exception('Unkown topology type: %s' % topo_type)
 
-    config_map = proto_dict(faucet_config)
+    config_map = cleanup_config(proto_dict(faucet_config))
```

Comment:
Can the fix be in proto_dict instead? Or for some situations, it’s necessary to have str keys?

Reply:
There are definitely cases where str keys are valid, so I can't think of any easy way to make it more general without being overly complicated (as in, dynamically investigating the proto to see what it's supposed to be and then patching it up).